sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -844,6 +852,35 @@ Exceptions:
 * Throws synchronously with error code `invalid` if the topology of `graph` is
   not the same as the existing graph topology, or if the nodes were not added in
   the same order.
+
+  |
+[source, c++]
+----
+using namespace ext::oneapi::experimental;
+template <typename T>
+void command_graph<graph_state::executable>
+update_memory_parameter(const T& from, const T& to);
+----
+
+|Updates a memory parameter across the executable graph such that all nodes in
+the graph which use `from` as a parameter will have that parameter replaced by
+`to`.
+
+Preconditions:
+
+* This member function is only available when the `command_graph` state is
+  `graph_state::executable`.
+* T must be either a buffer or a raw pointer to a USM allocation.
+
+Parameters:
+
+* `from` - The parameter within the graph to be updated.
+* `to` - The new value to use for this parameter.
+
+Exceptions:
+
+* Throws synchronously with error code `invalid` if `from` is not found within the graph
+or if `to` is not compatible with `from`.
```

Comment:
Do we need to specify what incompatible entails here? I feel like this statement is very generic.